tests/terraform/checks/resource/aws/test_MQBrokerNotPubliclyExposed.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+
+from checkov.terraform.checks.resource.aws.MQBrokerNotPubliclyExposed import check
+from checkov.common.models.enums import CheckResult
+
+
+class TestMQBrokerNotPubliclyExposed(unittest.TestCase):
+
+    def test_failure_mqbroker_logging(self):
+        resource_conf = {
```

Comment:
@mattfenwick To ensure the conf is configured correctly, try to write the matching HCL block and then parse it using hcl2. [Example for this pattern](https://github.com/bridgecrewio/checkov/blob/ddb9b8eca0774796ba44d29f0460fe4576b69792/tests/terraform/checks/resource/aws/test_IAMRoleAllowsPublicAssume.py#L11)

Reply:
👍